tests/shutdown_test.cpp

```diff
@@ -0,0 +1,79 @@
+#include <appbase/application.hpp>
```

Comment:
You added this file as a test but didn't plumb it through cmake etc. not sure if that's intentional or not

Reply:
`tests/CMakeFiles.txt` has `file(GLOB UNIT_TESTS "*.cpp")`

Reply:
I completely missed the glob :+1: